files/es/web/html/microdata/index.md

```diff
@@ -67,19 +67,19 @@ _[itemref](/es/docs/Web/HTML/Global_attributes/itemref)_: las propiedades que no
 
 ```html
 <div itemscope itemtype="http://schema.org/SoftwareApplication">
-  <span itemprop="name">Angry Birds</span> -
```

Comment:
These also don't look correct, as now the spans aren't closed (and it's in an example block that Markdownlint shouldn't be complaining about)